src/sso/__snapshots__/sso.spec.ts.snap

```diff
@@ -6,14 +6,14 @@ exports[`SSO SSO getAuthorizationURL with no custom api hostname generates an au
 
 exports[`SSO SSO getAuthorizationURL with state generates an authorize url with the provided state 1`] = `"https://api.workos.com/sso/authorize?client_id=proj_123&domain=lyft.com&redirect_uri=example.com%2Fsso%2Fworkos%2Fcallback&response_type=code&state=custom%20state"`;
 
-exports[`SSO SSO getProfile with all information provided sends a request to the WorkOS api for a profile 1`] = `"https://api.workos.com/sso/token?client_id=proj_123&client_secret=sk_test_Sz3IQjepeSWaI4cMS4ms4sMuU&code=authorization_code&grant_type=authorization_code&redirect_uri=https%3A%2F%2Fexmaple.com%2Fsso%2Fworkos%2Fcallback"`;
+exports[`SSO SSO getProfile with all information provided sends a request to the WorkOS api for a profile 1`] = `"https://api.workos.com/sso/token?client_id=proj_123&client_secret=sk_test_Sz3IQjepeSWaI4cMS4ms4sMuU&redirect_uri=https%3A%2F%2Fexmaple.com%2Fsso%2Fworkos%2Fcallback&grant_type=authorization_code&code=authorization_code"`;
 
 exports[`SSO SSO getProfile with all information provided sends a request to the WorkOS api for a profile 2`] = `
 Object {
-  "headers": Object {
-    "Authorization": "Bearer sk_test_Sz3IQjepeSWaI4cMS4ms4sMuU",
-    "User-Agent": "workos-js/0.0.3",
-  },
+  "Accept": "application/json, text/plain, */*",
```

Comment:
seems weird these aren't in a headers object anymore?

Reply:
I had to use `mock-axios-adapter` because a single `AxiosInstance` is created in the main constructor now so accessing the headers is different.